src/core/model/table/column.h

```diff
@@ -16,10 +16,8 @@ class Column {
     RelationalSchema const* schema_;
 
 public:
-    Column(RelationalSchema const* schema, std::string name, int index) :
-        name_(std::move(name)),
-        index_(index),
-        schema_(schema) {}
+    Column(RelationalSchema const* schema, std::string name, size_t index)
+        : name_(std::move(name)), index_(index), schema_(schema) {}
```

Comment:
If we're going down the path of replacing `int` for indices with `size_t`, we should also create an alias for this type  to make next such refactoring much easier (and the code more readable). 
Also, almost any algorithm works with columns and represents them using indices of some type, ideally these types should all be replaced with the introduced alias. However, it may be a lot of work to do, so I think it's acceptable to write a TODO comment/create an issue about this.

Reply:
BTW, there is already an alias for this in `src/core/algorithms/fd/hycommon/types.h`, probably we can extract it and reuse here

Reply:
I meant that reusing the alias in all algorithms can be postponed, introducing the alias IMHO should be done in this PR along with changing the index type from `int` to `size_t`. Creating something like `Column::IndexType = size_t` and using it instead of `int` for `Column` index is simple enough, but still eases the next refactoring of rewriting all algorithms to use it